game/scripts/vscripts/items/charge_bkb.lua

```diff
@@ -97,3 +97,7 @@ function modifier_charge_bkb_on_created(keys)
 --      return 10.0
 --   end)
 end
+
+item_charge_bkb_2 = item_charge_bkb
```

Comment:
at this point I have my doubts if anyone does, then.

...is a lua rewrite in order?